src/lib/game/handler.js

```diff
@@ -27,9 +27,10 @@ class GameHandler extends Socket {
   }
 
   // Connects to given host through given port
-  connect(host, port) {
+  connect(host, realm) {
+    this.realm=realm;
```

Comment:
Could you add spaces around operators, adhering to the [styleguide](http://eslint.org/docs/rules/space-infix-ops)?